puppet/modules/jenkins_job_builder/files/centos.org/ansible/deprovision.yml

```diff
@@ -0,0 +1,11 @@
+---
+- hosts: localhost
+  vars:
+    cico_data: "{{ lookup('file', './cico_data.json') | from_json }}"
+    api_key: "{{ lookup('file', '~/duffy.key') }}"
+  tasks:
+    - name: 'Destroy node'
+      cico:
+        action: done
+        api_key: "{{ api_key }}"
+        ssid: cico_ data.ssid
```

Comment:
Do I need :eyeglasses: or is there a space between `_` and `data`?

Reply:
Good find, odd that this did not error